priv/posts/marketing/20250624023712_getting-to-know-us-jonatan-mannchen.md

```diff
@@ -0,0 +1,87 @@
+{ "title": "Getting to know - Jonatan Männchen", "authors": ["The Marketing Working Group"], "slug": "getting-to-know-jonatan-mannchen", "category": "marketing", "tags": ["developer", "marketing", "getting-to-know-us", "bio"], "datetime": "2025-06-24T02:37:12.810583Z" }

```

Comment:
This should be multi line vs all on a single line see [here](https://github.com/erlef/website/blob/main/priv/posts/marketing/20230316023712_getting-to-know-bram-verburg.md?plain=1) for an example of how this should be done.